slurm/slurm.sh

```diff
@@ -0,0 +1,300 @@
+# Submit a job to the SLURM cluster
+
+slurm_submit(){
+    local commandString="$1"
+    local jobQueue="$2"
+    local jobName="$3"
+    local slurmMem="$4"
+    local nThreads="$5"
+    local jobGroupName="$6"       
+    local workingDir="$7"
+    local logPrefix="$8"
+    local prioritise="$9"
+    local condaEnv="${10}"
+    local quiet="${11:-'no'}"
+
+    # Need at least the command string
+
+    if [ -z "$commandString" ]; then
+        die "Need at least a command string for SLURM submission"
+    fi
+
+    # Check parameter settings
+
+    if [ -n "$jobQueue" ]; then jobQueue=" -p ${jobQueue}"; fi
+    if [ -n "$jobName" ]; then jobName=" --J ${jobName}"; fi
+    if [ -n "$slurmMem" ]; then slurmMem=" --mem $slurmMem"; fi
+    if [ -n "$nThreads" ]; then nThreads=" --cpus-per-task $nThreads"; fi
+    if [ -n "$jobGroupName" ]; then jobGroupName=" --job-group $jobGroupName"; fi
+    if [ -n "$workingDir" ]; then workingDir=" --chdir \"$workingDir\""; fi
+    if [ -n "$condaEnv" ]; then
+        condaBase=$(conda info --json | awk '/conda_prefix/ { gsub(/"|,/, "", $2); print $2 }')
+        #condaCmd=". ${condaBase}/bin/activate ${condaBase}/envs/${condaEnv}"
+        ###### the lines below were added to enable the activation of a conda env that is not inside the base conda env directory
+        condaEnvPath="${condaBase}/envs/${condaEnv}"
+        if [ -d "$condaEnv" ]; then condaEnvPath="$condaEnv"; fi
+        condaCmd=". ${condaBase}/bin/activate ${condaEnvPath}"
+        ######
+        commandString="${condaCmd} && ${commandString}"
+    fi
+    if [ -n "$logPrefix" ]; then 
+        mkdir -p $(dirname $logPrefix)
+        logPrefix=" -o \"${logPrefix}.out\" -e \"${logPrefix}.err\""
+    fi
+    maxTime=$(slurm_maxtime_for_partition "production")
```

Comment:
I think here instead of "production" , this queue (partition) should come from `atlas-slurm`

```
-q <slurm queue, defaults to cluster default
```

Reply:
I think this is solved by your code